homepage/models.py

```diff
@@ -322,6 +322,23 @@ def get_courses_followed_by_app_user(app_user):
         # get only 'course' elements from user_course_pair elements
         return [user_course_pair.course for user_course_pair in pairs]
 
+    @classmethod
+    def is_following_course(cls, user, course):
+        app_user = user.appuser
+        return True if course in cls.get_courses_followed_by_app_user(app_user) else False

```

Comment:
if you use app_user only once and it is not complicated to retrieve this value 
I think you can use it directly in the next line and give up on this line
```suggestion
        return course in cls.get_courses_followed_by_app_user(user.appuser)

```

Reply:
Changed all occurrences